packages/components/src/components/form/bal-select/bal-select.tsx

```diff
@@ -723,7 +721,7 @@ export class Select {
   }
 
   private syncNativeInput() {
-    if (!this.multiple && !this.remote) {
+    if (!this.multiple) {
```

Comment:
shouldn't we also sync if it is multiple? is it done as well ?

Reply:
because the value is not presented in the native input for multiple